website/contributing/contributing-overview.md

```diff
@@ -103,17 +103,18 @@ Directly in the repo, there is the [`rn-tester` app](https://github.com/facebook
 The process of proposing a change to React Native can be summarized as follows:
 
 1. Fork the React Native repository and create your branch from `main`.
-2. Make the desired changes to React Native sources. Use the `packages/rn-tester` app to test them out.
-3. If you've added code that should be tested, add tests.
-4. If you've changed APIs, update the documentation, which lives in [separate repo](https://github.com/facebook/react-native-website/).
-5. Ensure the test suite passes, either locally or on CI once you opened a pull request.
-6. Make sure your code lints (for example via `yarn lint --fix`).
-7. Push the changes to your fork.
-8. Create a pull request to the React Native repository.
-9. Review and address comments on your pull request.
-10. A bot may comment with suggestions. Generally we ask you to resolve these first before a maintainer will review your code.
-11. If changes are requested and addressed, please [request review](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify reviewers to take another look.
-12. If you haven't already, please complete the [Contributor License Agreement](/contributing/contribution-license-agreement) ("CLA"). **[Complete your CLA here.](https://code.facebook.com/cla)**
+2. For editing files in Android Studio, open the root react-native folder.
```

Comment:
```suggestion
2. For editing files in Android Studio, open the **root** react-native folder inside your IDE of choice (Visual Studio, Android Studio, Xcode).
```

Also @cipolleschi do you confirm that the root folder is correct for iOS?

Reply:
nope... for iOS you have to open the `RNTesterPod.xcworkspace` that is generated by Cocoapods.

Also, the sentence is `For editing files in Android Studio`... so... i would actually remove the suggested `inside your IDE of choice` because it looks weird otherwise! 😅